packages/composite-checkout/src/types.ts

```diff
@@ -244,6 +244,7 @@ export interface TransactionStatusManager extends TransactionStatusState {
 	setTransactionComplete: SetTransactionComplete;
 	setTransactionPending: SetTransactionPending;
 	setTransactionRedirecting: SetTransactionRedirecting;
+	dispatchTransactionStatusAction: Dispatch< TransactionStatusAction >;
```

Comment:
This should be removed because it exposes an implementation detail of the package.